helm/aws-efs-csi-driver-bundle/templates/_helpers.tpl

```diff
@@ -0,0 +1,84 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "aws-efs-csi-driver-bundle.name" -}}
+{{- default .Chart.Name .Values.bundleNameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "aws-efs-csi-driver-bundle.fullname" -}}
+{{- if .Values.fullBundleNameOverride -}}
+{{- .Values.fullBundleNameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.bundleNameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "aws-efs-csi-driver-bundle.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Common labels
+*/}}
+{{- define "aws-efs-csi-driver-bundle.labels" -}}
+app.kubernetes.io/name: {{ include "aws-efs-csi-driver-bundle.name" . }}
+helm.sh/chart: {{ include "aws-efs-csi-driver-bundle.chart" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+giantswarm.io/service-type: "managed"
+application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
+giantswarm.io/cluster: {{ .Values.clusterID | quote }}
+{{- end -}}
+
+{{/*
+Create a string out of the map for controller tags flag
+*/}}
+{{- define "aws-efs-csi-driver-bundle.tags" -}}
+{{- $tags := list -}}
+{{ range $key, $val := . }}
+{{- $tags = print $key ":" $val | append $tags -}}
+{{- end -}}
+{{- join " " $tags -}}
+{{- end -}}
+
+{{/*
+Get trust policy statements for all provided OIDC domains
+*/}}
+{{- define "trustPolicyStatements" -}}
+{{- $configmap := (lookup "v1" "ConfigMap" .Release.Namespace (printf "%s-crossplane-config" .Values.clusterID)) -}}
+{{- $cmvalues := dict -}}
+{{- if and $configmap $configmap.data $configmap.data.values -}}
+  {{- $cmvalues = fromYaml $configmap.data.values -}}
+{{- end -}}
+{{- range $index, $oidcDomain := $cmvalues.oidcDomains -}}
+{{- if not (eq $index 0) }}, {{ end }}{
+  "Effect": "Allow",
+  "Principal": {
+    "Federated": "arn:{{ $cmvalues.awsPartition }}:iam::{{ $cmvalues.accountID }}:oidc-provider/{{ $oidcDomain }}"
+  },
+  "Action": "sts:AssumeRoleWithWebIdentity",
+  "Condition": {
+    "StringLike": {
+      "{{ $oidcDomain }}:sub": "system:serviceaccount:kube-system:aws-efs-csi-driver"
```

Comment:
Should this use `{{ .Values.controller.serviceAccount.name }}` instead of hardcoding?